UN-OCHA/unocha-site
# UNO-635 Donors

UNO-635 Library, styles, template with markup and hardcoded values for Donors, split into:

1. Top Donors and Top Funded Sectors
2. Top Donors for CBPF
3. Earmarked and Unearmarked for OCT

All except Unearmarked (which doesn't display the value, just the label) is a `dl`. Unearmarked is a `ul`

Paragraph template override `paragraph--donors.html.twig` includes the partial `donors.html.twig` but note I did not include the config to create a paragraph type because I don't know how we will choose to implement this.

More info on UNO-635

### To test:
Set up a paragraph type called "Donors" and add an instance to a Response page underneath the set of tabs (it doesn't really matter right now where it goes) or import the partial into whatever template you wish (Eg. `node--response.html.twig`) so you can see it on the frontend.
